query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/scalar_list/json.rs

```diff
@@ -161,8 +162,9 @@ mod json {
         // TODO: This specific query currently cannot be sent from the JS client.
         // The client _always_ sends an array as plain json and never as an array of json.
         // We're temporarily ignoring it for the JSON protocol because we can't differentiate a list of json values from a json array.
+        // Similarly, this does not currently work with driver adapters.
```

Comment:
Could you elaborate on why this doesn't work with external executors at all?

Reply:
I'm going to merge, but please @aqrln get back to this question when you have time. I am also not sure why is this the case.

Reply:
Um, actually I'm not sure anymore, it seemed "obvious" to me two days ago but now I'm also confused. I can take another look, or we can un-skip this for now so it fails together with other tests we haven't fixed or investigated yet. WDYT?